Src/Base/AMReX_FileSystem.cpp

```diff
@@ -82,7 +82,7 @@ CreateDirectories (std::string const& path, mode_t mode, bool verbose)
     bool retVal(false);
     Vector<std::pair<std::string, int> > pathError;
 
-    const char* path_sep_str = "/";
+    const char* path_sep_str = DirectorySeparator();
```

Comment:
Not needed and I just want the new universal function for general use.
```suggestion
    const char* path_sep_str = "/";
```